circle.yml

```diff
@@ -50,6 +52,15 @@ dependencies:
     # Clean up any old images and save the new one
     - I="image-$(date +%j).tgz"; mkdir -p ~/docker; rm ~/docker/*; docker save app:build | gzip -c > ~/docker/$I; ls -l ~/docker
 
+    # build addon
```

Comment:
This stuff needs to go before [line 49](https://github.com/meandavejustice/testpilot/blob/447-circleci-addon/circle.yml#L49) where `docker build` happens, so the addon gets bundled into the container